test/libsolidity/semanticTests/reverts/error_struct.sol

```diff
@@ -1,17 +1,10 @@
-struct error { uint error; }
 contract C {
 	error test();
-	error _struct;
```

Comment:
The fact that a test case named `error_struct.sol` no longer contains an `error` struct makes it somewhat pointless ;)

Reply:
Agree, this could be removed.

Reply:
Or we could turn it into a negative syntax test that makes sure that this generates an error.